src/ds/d500/d500-color.cpp

```diff
@@ -165,6 +165,12 @@ namespace librealsense
         color_ep.register_metadata(RS2_FRAME_METADATA_AUTO_EXPOSURE, make_attribute_parser(&md_rgb_control::ae_mode, md_rgb_control_attributes::ae_mode_attribute, md_prop_offset,
             [](rs2_metadata_type param) { return (param != 1); })); // OFF value via UVC is 1 (ON is 8)
 
+        md_prop_offset = metadata_raw_mode_offset +
+            offsetof(md_rgb_mode, rgb_mode) +
+            offsetof(md_rgb_normal_mode, intel_capture_stats);
+
+        color_ep.register_metadata(RS2_FRAME_METADATA_FRAME_TIMESTAMP, make_attribute_parser(&md_capture_stats::hw_timestamp, md_capture_stat_attributes::hw_timestamp_attribute, md_prop_offset));
```

Comment:
Hey, before we merge, 1 question
why only color? IMO HKR will set this for all streams (depth, safety,occ,lpc)
Please make sure with Xuetao

Can you make sure with a drop from HKR that this is enabled?
Merging this will cause no timetag right?

Reply:
IMO we also need to change sensor TS to be the new HW TS - Sensor timestamp from MD which is 1/2 of the exposure

Reply:
Hi Nir - this is for merging to dev branch. I will also do the change for depth.

Reply:
regarding the sensor ts - this is the only one that remains as uvc header's ts, so I'd rather let it as is - do you agree?
Moreover, I need this in my test - see in this PR.

Reply:
On D400 and on D500 until now, uvc header timestamp was acctually the hw timestamp, fw set it to the time right after it grabbed the frame, and sensor ts reduced half the exposure so it was actually the sensor time stamp.
Now that FW will set the uvc header as the time the UVC message is constructed (could be much after the readout, reducing the exposure time has no meaning
This is why it needs to change to hw_timestamp - half the exposure (sensor timestamp in MD)
Please try to get more info from Evgeni on Sunday if needed